Ark.Tools.NLog/NlogConfigurer.cs

```diff
@@ -30,11 +31,20 @@ public static class NLogConfigurer
 
         static NLogConfigurer()
         {
-
             ConfigurationItemFactory.Default.RegisterItemsFromAssembly(typeof(Configurer).Assembly);
             ConfigurationItemFactory.Default.RegisterItemsFromAssembly(typeof(ActivityTraceLayoutRenderer).Assembly);
             LogManager.LogFactory.ServiceRepository.RegisterService(typeof(IJsonConverter), new STJSerializer());
 
+            InternalLogger.LogLevel = LogLevel.Warn;
+            InternalLogger.LogToConsole = true;
+
+            var appName = AppDomain.CurrentDomain.FriendlyName ?? Assembly.GetEntryAssembly()?.GetName().Name ?? "Unknown";

```

Comment:
Comment here why this is done.

This has been added support NLog loggers output to Console **during** application initialization, before Configuration is Read and Host is Built.